### PR TITLE
adding get product v3 method into content service

### PR DESCRIPTION
**Pull request by PetrStoklas**
## What does this PR do & what background information is important for this PR?
Adding get product v3 method into content service

## Checklist
- [ ] Updated documentation (if required, see README.md)
- [ ] Ran locally: `npm run lint && npm run test-coverage`
- [ ] Bumped version on package.json

### CODE REVIEW

File: src/content-service/services/api-provider.ts [hunk 1]
```diff
@@ -57,6 +57,18 @@ export const getContentServiceApi = (
     return data;
   };
 
+  const getProductFromV3 = async (id: string): Promise<ApiProductDataV3> => {
+    const requestUrl = `${productsPath}/${id}?getContentFromV3=true`;
```
Comment (leonardomorenozaga) [inline src/content-service/services/api-provider.ts:61]: this path need to be adjusted to hit the right version of the service:
`https://content-service.todaytix.com/api/v3/products/34579?includeAllFields=true'

Also we can remove `getContentFromV3` param and add `includeAllFields=true` instead.

Comment (PetrStoklas) [reply src/content-service/services/api-provider.ts]: pushed new code, pls have a look if I am in the right direction

File: src/content-service/services/repository-provider.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { Product } from '../models';
 import { getContentServiceApi } from './api-provider';
 import { checkRequiredProperty } from '../../utils/validator';
 import { Environment, Settings, SourceInformation } from '../../shared/typings';
+import { ProductV3 } from "../models/productV3";
```
Comment (leonardomorenozaga) [inline src/content-service/services/repository-provider.ts:5]:
```suggestion
import { ProductV3 } from '../models/productV3';
```

File: src/content-service/typings/index.ts [hunk 3]
```diff
@@ -40,6 +45,20 @@ export interface ApiAddressData {
   country: ApiCountryData;
 }
 
+export interface ChartDetails {
```
Comment (leonardomorenozaga) [inline src/content-service/typings/index.ts:48]: where is this one used?

**Review by leonardomorenozaga — APPROVED** [review]
looks good, just  a final comment about single quotes

**Pull request merged**